examples/hacker_news_assets/hacker_news_assets/jobs/hacker_news_api_download.py

```diff
@@ -16,34 +16,8 @@
 }
 
 
-ASSETS = [id_range_for_time, items, comments, stories]
+from ..asset_collection import asset_collection
 
+hn_client = hn_api_subsample_client.configured({"sample_rate": 10})
 
-download_prod_job = build_assets_job(
-    "hacker_news_api_download",
-    assets=ASSETS,
-    resource_defs={
-        **{"hn_client": hn_api_subsample_client.configured({"sample_rate": 10})},
-        **RESOURCES_PROD,
-    },
-    tags=DOWNLOAD_TAGS,
-)
-
-
-download_staging_job = build_assets_job(
-    "hacker_news_api_download",
-    assets=ASSETS,
-    resource_defs={
-        **{"hn_client": hn_api_subsample_client.configured({"sample_rate": 10})},
-        **RESOURCES_STAGING,
-    },
-    tags=DOWNLOAD_TAGS,
-)
-
-download_local_job = build_assets_job(
-    "hacker_news_api_download",
-    assets=ASSETS,
-    resource_defs={**{"hn_client": hn_snapshot_client}, **RESOURCES_LOCAL},
-    tags=DOWNLOAD_TAGS,
-    executor_def=in_process_executor,
-)
+download_job = asset_collection.build_asset_job(subset="id_range_for_time*", tags=DOWNLOAD_TAGS)
```

Comment:
An interesting thing here is that this job is resource-less. What are the situations where we'd want to use this object?

Reply:
My thinking was that if the resources are on the repository, there's no need to have resources on the job as well. I'd imagine you might want to use the same job in multiple repos, similar to how you do now, and this makes it easier to do so.